app/lib/ui/views/account/account_editing_screen.dart

```diff
@@ -0,0 +1,187 @@
+import 'package:app/core/models/transporter.dart';
+import 'package:app/core/view_models/account_edit_view_model.dart';
+import 'package:app/ui/common/view_state.dart';
+import 'package:app/ui/widgets/utility/busy_overlay_screen.dart';
+import 'package:app/ui/widgets/utility/template_base_view_model.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_multi_formatter/flutter_multi_formatter.dart';
```

Comment:
I don't see this being used anywhere?

Reply:
multi fromatter is used to format the phone number. For instance, '+1 (306) 123 456'. This format is based on our prototype design

Reply:
Oh, my bad. I see it now. We should use this in the sign up screen as well